demo/rviz.launch.py

```diff
@@ -52,7 +52,7 @@ def generate_launch_description():
         [
             DeclareLaunchArgument(
                 "robot_namespace",
-                default_value=EnvironmentVariable("ROS_NAMESPACE", default_value=""),
+                default_value=EnvironmentVariable("ROBOT_namespace", default_value=""),
```

Comment:
```suggestion
                default_value=EnvironmentVariable("ROBOT_NAMESPACE", default_value=""),
```
I think the global env should be written upper case.